src/DisplayTypes.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+module DisplayTypes where
+
+import GHC.Generics
+import Data.Aeson 
+
+
+-- |
+-- Displayable takes a type and wraps it in a Display
+-- Display will be used in HaskellDO to handle media rendering
+class Displayable a where
+  display :: a -> Display
+
+-- |
+-- DisplayType contains the allowed media to render 
+-- Any extensions to this library (i.e custom Display instances) should add a type here 
+-- E.g adding a display for latex would add the type DisplayLatex here
+data DisplayType
```

Comment:
I think that this should not be that "open", we should have just:

- Text
- HTML
- SVG
- JS
- Image

We can add more types in the future, but we should not constraint users into adding their instance here.